src/main/java/seedu/address/model/transaction/Budget.java

```diff
@@ -57,11 +82,46 @@ private void updateDeadline(Date date) {
     }
 
     /**
-     * Calculates the new Date given the amount of duration from Today
+     * Calculates the new Date given the amount of duration from Today.
      * @return Date after {@code duration} days from today
      */
     private Date calculateDeadline(int duration) {
```

Comment:
@yewon0303 I think should create the necessary methods in the Date class instead of using LocalDate directly. Kinda breaks abstraction here.